installer/scripts/onboard_agent.sh

```diff
@@ -6,10 +6,10 @@
 
 
 # Values to be updated upon each new release
-GITHUB_RELEASE_X64="https://github.com/microsoft/OMS-Agent-for-Linux/releases/download/OMSAgent_v1.13.40-0/"
+GITHUB_RELEASE_X64="https://github.com/microsoft/OMS-Agent-for-Linux/releases/download/OMSAgent_v1.14.1-0/"
```

Comment:
again, why tag 1.14.1?

Reply:
initially I started the release as 1.14.1, and it becomes 1.14.9, I fixed it.

Reply:
When we run wget https://raw.githubusercontent.com/Microsoft/OMS-Agent-for-Linux/master/installer/scripts/onboard_agent.sh && sh onboard_agent.sh -w <YOUR WORKSPACE ID> -s <YOUR WORKSPACE PRIMARY KEY> on our Linux machines it installs 1.13.40. Isn't expected behavior that it upgrades to 1.14.x?